extensions/kubernetes/kind/deployment/src/main/resources/META-INF/services/io.dekorate.ManifestGeneratorFactory

```diff
@@ -0,0 +1 @@
+io.dekorate.kind.manifest.KindManifestGeneratorFactory
```

Comment:
I think that this needs to be addressed on dekorate.

Reply:
+1

Reply:
So should we merge this one or is there some additional work planned we want to push to this PR?

Reply:
No, the PR is ready to be merged. 
Take into account that this PR depends on Fabric8 Kubernetes Client 6.x which is only in main.